README.md Update (#47)
snek-at/intel: README.md

<h3 align="center">SNEKNetwork for Engineers and Knowledged</h3>
<p align="center">

> The SNEK project is an attempt to create a transparent, open-source non-profit platform that allows engineers to categorize and compare. It should enable engineers from adjacent fields to visualize each other's skills through visualization and project identification.

Reviewer: Missing comma bro.
```suggestion
The SNEK project is an attempt to create a transparent, open-source, non-profit platform that allows engineers to categorize and compare. It should enable engineers from adjacent fields to visualize each other's skills through visualization and project identification.
```

Reviewer: Nevermind. I was proven wrong here brotha.

<h3 align="center">SNEKNetwork for Engineers and Knowledged</h3>
<p align="center">

> The SNEK project is an attempt to create a transparent, open-source, non-profit platform that enables engineers to categorize and compare themselves with other software developers in their respective fields, through project identification and visialization.

Reviewer 2: I think the old description was better.

Reviewer 2: ```suggestion
The SNEK project is an attempt to create a transparent, open-source non-profit platform that allows engineers to categorize and compare. It should enable engineers from adjacent fields to visualize each other's skills through visualization and project identification.
```

## [](#contributing)Contributing
![GitHub last commit](https://img.shields.io/github/last-commit/snek-at/intel) ![GitHub issues](https://img.shields.io/github/issues-raw/snek-at/intel) ![GitHub closed issues](https://img.shields.io/github/issues-closed-raw/snek-at/intel?color=green)

> Please read through our [guidelines for contribution](https://github.com/snek-at/front/blob/master/CONTRIBUTING.md). Included are directions for opening issues, coding standards, and notes on development.

Reviewer 3: I would change this back to "contributing guidelines"

Reviewer 2: fair

Reviewer 2: ```suggestion
Please read through our [contributing guidelines](https://github.com/snek-at/front/blob/master/CONTRIBUTING.md). Included are directions for opening issues, coding standards, and notes on development.
```
